security/pkg/pki/util/keycertbundle.go

```diff
@@ -254,29 +254,29 @@ func (b *KeyCertBundle) UpdateVerifiedKeyCertBundleFromFile(certFile string, pri
 }
 
 // ExtractRootCertExpiryTimestamp returns the unix timestamp when the root becomes expires.
-func (b *KeyCertBundle) ExtractRootCertExpiryTimestamp() (float64, error) {
+func (b *KeyCertBundle) ExtractRootCertExpiryTimestamp() (*time.Time, error) {
 	return extractCertExpiryTimestamp("root cert", b.GetRootCertPem())
 }
 
 // ExtractCACertExpiryTimestamp returns the unix timestamp when the cert chain becomes expires.
-func (b *KeyCertBundle) ExtractCACertExpiryTimestamp() (float64, error) {
+func (b *KeyCertBundle) ExtractCACertExpiryTimestamp() (*time.Time, error) {
 	return extractCertExpiryTimestamp("CA cert", b.GetCertChainPem())
 }
 
 // TimeBeforeCertExpires returns the time duration before the cert gets expired.
 // It returns an error if it failed to extract the cert expiration timestamp.
-// The returned time duration could be a negative value indicating the cert has already been expired.
```

Comment:
I'm pretty sure this wasn't true.
Before, it the certificate had expired, `extractCertExpiryTimestamp` would return both the expiration timestamp and an error, so in line 275, this functions would return 0, which isn't negative